examples/gg20_sign_client.rs

```diff
@@ -170,6 +170,11 @@ fn main() {
             j += 1;
         }
     }
+
+    assert_eq!(signers_vec.len(), (THRESHOLD + 1) as usize);
+    if signers_vec.len()> (THRESHOLD + 1) as usize{
```

Comment:
I think the assert_eq! can just be removed and the if can be changed to:
```
if signers_vec.len() != (THRESHOLD + 1) as usize{
panic!("For the demo we always sign with minimum number of parties(THRESHOLD + 1). This should not happen.");
}
```

Reply:
Why **minimum** number of parties, shouldn't it be exactly THRESHOLD + 1?

Reply:
that's what I meant by minimum number of parties. You need minimum THRESHOLD + 1 signers to generate a usable signature.

Also this comment is just for style. We don't need an assert and an if statement. Could just use an if statement only.

But that should not stop you from merging this as is.